python/packages/ai/teams/ai/models/openai_model.py

```diff
@@ -124,11 +125,47 @@ async def complete_prompt(
         template: PromptTemplate,
     ) -> PromptResponse[str]:
         max_tokens = template.config.completion.max_input_tokens
+
+        # Setup actions if enabled
+        is_tools_aug = (
+            template.config.augmentation
+            and template.config.augmentation.augmentation_type == "tools"
+        )
+        tool_choice = (
```

Comment:
im confused by this being configurable in the `config.json` but also a temp state variable.

Reply:
yup it's defined by the developer in the config.json but we have it saved in temp state bc we don't have access to the template in tools_augmentation